pages/search/index.vue

```diff
@@ -104,6 +104,16 @@
               />
             </b-col>
           </b-row>
+          <b-row>
+            <b-col>
+              <TierToggler
+                :text="tier.text"
+                :button="tier.button"
+                :toggled="tierToggle"
```

Comment:
don't use `tierToggle` here, rather get the value from the selectedFacets. Perhaps in a computed method if it's too complex to just pull out of the selectedFacets. It'll also only want to be true if the selected value is actually `*`.